app/models/manageiq/providers/openshift/container_manager/options.rb

```diff
@@ -0,0 +1,4 @@
+module ManageIQ::Providers::Openshift::ContainerManager::Options
+  extend ActiveSupport::Concern
```

Comment:
:+1: without this, the ClassMethods would be available as class methods on this module, but not on Openshift::ContainerManager that includes it.  :sob: :laughing: oh well ruby is weird sometimes...